neverbleed.c

```diff
@@ -45,6 +45,10 @@
 #endif
 #include "neverbleed.h"
 
+#define OPENSSL_101_API (!defined(LIBRESSL_VERSION_NUMBER) && OPENSSL_VERSION_NUMBER >= 0x1010000fL)
```

Comment:
How about renaming this to `OPENSSL_1_1_API`. IMO, `101` is confusing with `1.0.1`.